frame/executive/src/lib.rs

```diff
@@ -145,9 +145,9 @@ impl<
 where
 	Block::Extrinsic: Checkable<Context> + Codec,
 	CheckedOf<Block::Extrinsic, Context>:
-		Applyable<DispatchInfo=DispatchInfo> +
+		Applyable +
 		GetDispatchInfo,
```

Comment:
`GetDispatchInfo` should be obsolete now, no? Or rather we should simply merge it with `Dispatchable`

Reply:
Not sure. need to pass it to `apply`, hence `Applyable` still needs to be `GetDispatchInfo`. I think this can be a separate PR, even if applicable.

Reply:
Yes I think we can merge it now into `Dispatchable`. I will look into this.

Reply:
I think it is a too big refactor itself. It should have its own PR one this is in.